src/components/pages/pages.stories.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import { Pages, Page } from './pages';
+import DialogFullScreen from '../dialog-full-screen/dialog-full-screen';
+import Heading from '../heading/heading';
+import Button from '../button/button';
+
+const store = new Store({
+  open: false,
+  slideIndex: 0
+});
+
+const handleSlide = (ev) => {
+  action('slide')();
```

Comment:
If you pass the `event` as an argument, you'll be able to see it / drill down into it in the actions panel.

Reply:
Fixed 👍